app/gui/view/welcome-screen/src/lib.rs

```diff
@@ -134,7 +139,15 @@ impl Model {
 // ===========
 
 ensogl::define_endpoints! {
-    Input {}
+    Input {
+        /// Update displayed list of projects.
+        projects_list(Vec<String>),
```

Comment:
this should be named `set_projects_list`

Reply:
Makes sense, done